tools/cli/help.txt

```diff
@@ -17,21 +17,26 @@ Built-in Commands:
 
 See 'meteor help <built-in command>' for details on a command.
 
+Other Commands:
+
 Meteor also bundles 'node' and 'npm' by default. If you use the
 commands 'meteor node' or 'meteor npm', you will be running
 the same versions of 'node' and 'npm' that Meteor uses internally.
 
-Npm commands like 'yarn', 'jsdoc', etc., can also be installed within
-Meteor itself and used within all apps using the same version.
-
-First install them in your Meteor global context with:
+Other commands can be installed in the same directory as 'node' and
+'npm' and used in all apps using the same version of Meteor. An easy
+way to install such commands is to 'meteor npm install --global' an
+npm package ('yarn', 'jsdoc', etc.) that provides the command:
```

Comment:
I would change `to install such commands is to 'meteor npm install --global' an npm package ('yarn', 'jsdoc', etc.) that provides the command`
with simply ` to install such commands is to add an npm package ('yarn', 'jsdoc', etc.) that provides the command with:`

The rest looks perfect to me 👍